mint.json

```diff
@@ -33,7 +33,7 @@
       ]
     },
     {
-      "group": "Unit",
+      "group": "Property",
```

Comment:
This might have come from Tripty or Andy, in which case ignore. Otherwise, might be worth thinking if "Property" is confusing. A property could be viewed as an entire apartment complex by our customers, within which there are multiple units.

Reply:
Andy asked for this nav item to be changed to "Property' from "Unit". I think some other endpoints are going to live under this "heading", so it'll make more sense later on.

Reply:
Cool, makes sense.